source/dsl/DangerDSL.ts

```diff
@@ -23,9 +23,8 @@ export interface DangerDSLType {
 export class DangerDSL {
   public readonly github: Readonly<GitHubDSL>
 
-  constructor(pr: GitHubPRDSL, public readonly git: GitDSL) {
-    this.github = {
-      pr
-    }
+  constructor(platformDSL: any, public readonly git: GitDSL) {
+    // As GitLab etc support is added this will need to be changed
+    this.github = platformDSL
   }
```

Comment:
it is now not the responsibility of this class to create the `github` object, that's moved to `getPlatformDSLRepresentation`. In the future maybe this should change, whoever is figuring out gitlab can figure that ;)